modules/offers/client/components/OfferLocationPresentational.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import L from 'leaflet';
+
+import { Map, TileLayer, Circle, Marker } from 'react-leaflet';
+
+export default function OfferLocationPresentational({ zoom, location, onChangeZoom, marker='', windowWidth }) {
+
+  return (
+    <Map
+      className="offer-location"
+      attributionControl={false}
+      zoom={zoom}
+      center={location}
+      onZoom={({ target: { _zoom: zoom } }) => onChangeZoom(zoom)}
+      scrollWheelZoom={false}
+    >
+      <TileLayer
+        attribution='&amp;copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      />
+      {/* @TODO Circle and Marker will need to be reusable when we migrate the /search to React */}
+      {(zoom >= 12) ?
+        <Circle
+          center={location}
+          radius={500}
+          weight={2}
+          color="#989898"
+          fillColor="#b1b1b1"
+          fillOpacity={0.5}
+          clickable={false}
+          interactive={false}
+          layer="locationPath"
+        /> :
+        <Marker
+          position={location}
+          layer="locationMarker"
+          clickable={false}
+          focus={false}
+          icon={getIcon(marker, windowWidth)}
+        />
+      }
+    </Map>
+  );
+}
+
+/**
+ * Get url of marker icon
+ */
+function getMarkerUrl(marker) {
+  const markerName = ['marker', 'icon', ...((marker) ? [marker] : [])].join('-');
+  return `/img/map/${markerName}.svg`;
```

Comment:
This feels unnecessarily complex, especially since `marker` travels through all these functions and components so that at the end it's really hard to tell what it contains. :-/

Would this help?
```js
/**
 * Get url of marker icon
 *
 * @param {String} marker - maybe, meet, yes or an empty string 
 */
function getMarkerUrl(marker) {
  const markerFile = marker ? `-${marker}` : '';
  return `/img/map/marker-icon${markerFile}.svg`;
}
```